### PR TITLE
expanded province names and added an extra default value

### PR DESCRIPTION
**Pull request by sakura-sketch**
expanded province names and added an extra default value in the drop down

### CODE REVIEW

**Comment by YedidaZalik** [comment]
Looks good in English.
Please add French translation.
@EliseKa please assist.
Merci!

File: register/forms.py [hunk 1]
```diff
@@ -63,19 +63,20 @@ class LocationNameForm(HealthcareBaseForm, forms.Form):
...
+    ("NB", "New Brunswick"),
+    ("NL", "Newfoundland and Labrador"),
+    ("NS", "Nova Scotia"),
+    ("NT", "Northwest Territories"),
+    ("NU", "Nunavut"),
+    ("ON", "Ontario"),
+    ("PE", "Prince Edward Island"),
+    ("QC", "Quebec"),
+    ("SK", "Saskatchewan"),
+    ("YT", "Yukon"),
```
Comment (dsamojlenko) [inline register/forms.py:79]: These should all be translated strings

File: locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -2036,6 +2036,64 @@ msgstr "Espaces religieux"
 msgid "Something else"
 msgstr "Autre"
 
+#: register/forms.py
+#, fuzzy
+#| msgid "Province or territory"
+msgid "Select a province or territory"
+msgstr "Province ou territoire"
```
Comment (dsamojlenko) [inline locale/fr/LC_MESSAGES/django.po:2043]: fuzzy match

**Comment by EliseKa** [comment]
Added the missing translation, also added the accents in the other provinces names.

**Review by dsamojlenko — APPROVED** [review]

**Pull request merged**